packages/ndla-article-scripts/src/conceptScripts.js

```diff
@@ -93,29 +93,33 @@ export const addShowConceptDefinitionClickListeners = () => {
 
         const parentOffset = getElementOffset(popup.offsetParent).top;
         const openBtnBottom = openBtn.getBoundingClientRect().bottom + window.pageYOffset - parentOffset;
-        popup.style.top = `${openBtnBottom - 500}px`;
+        popup.style.top = `${openBtnBottom - 200}px`;
         const viewportHeight = Math.max(document.documentElement.clientHeight, window.innerHeight || 0);
         const popupTop = getElementOffset(popup).top;
         const popupHeight = popup.offsetHeight;
-        let offset = 0;
+        let scrollOffset = 0;
+
+        let mastheadHeight = 84;
```

Comment:
Bør denne hardkodes? Kva når masthead har alerts?

Reply:
Dette håndteres rett under. Skulle aller helst brukt util-funksjonen for henting av mastheadHeight i ndla-ui, men dessverre er ndla-ui avhengig av article-scripts og dermed kan ikke ndla-ui importeres hit.